patches/bug-reporting/matrix-react-sdk+3.95.0.patch

```diff
@@ -42,46 +41,32 @@ index 5d826f2..51baf28 100644
 -            sendLogs: true,
 -            progressCallback: this.sendProgressCallback,
 -            labels: this.props.label ? [this.props.label] : [],
-+        // :TCHAP: customise report : add email, prefix with "tchap-web"
-+        const client = MatrixClientPeg.get();
-+        const customFields = {};
-+        client.getThreePids().then(result => {
-+            result.threepids.forEach(threepid => {
-+                return customFields[threepid.medium] = threepid.address;
-+            });
-+            return customFields;
-+        }).then(customFields => {
-+            // is this a voip report ? Add it in "context" field
++        // :TCHAP: add custom fields if it's a voip report
++        Promise.resolve().then(() => {
 +            if (this.props.label === "voip-feedback") {
++                const customFields = {};
 +                customFields.context = "voip";
-+            }
 +
-+            return TchapUtils.isCurrentlyUsingBluetooth().then(isCurrentlyUsingBluetooth => {
-+                customFields.audio_input = isCurrentlyUsingBluetooth ? "headset_bluetooth" : "device";
-+                return customFields;
-+            })
++                return TchapUtils.isCurrentlyUsingBluetooth().then(isCurrentlyUsingBluetooth => {
++                    customFields.audio_input = isCurrentlyUsingBluetooth ? "headset_bluetooth" : "device";
++                    return customFields;
++                });
++            } else {
++                return {};
++            }
```

Comment:
Some syntax suggestions with early return, to avoid one more depth in the code :
```
        if (this.props.label !== "voip-feedback") {
            return Promise.resolve({});
        }
        return TchapUtils.isCurrentlyUsingBluetooth().then(isCurrentlyUsingBluetooth => ({
            audio_input: isCurrentlyUsingBluetooth ? "headset_bluetooth" : "device",
            context: "voip"
        }))
```